src/map/map.ts

```diff
@@ -774,6 +776,16 @@ export class PropertiesMap {
         return this._selectTrace<number[]>(values, selected, trace);
     }
 
+    private _title(name: string): string {
+        if (name !== '') {
+            const units = this._property(name).units;
+            if (units !== undefined) {
+                return name + ` [${units}]`;
```

Comment:
should this be `${name)/${units}` instead, per https://github.com/cosmo-epfl/chemiscope/pull/88#issuecomment-716748562?

Reply:
I missed that comment. Let's do that!